packages/react-core/src/components/MenuToggle/examples/MenuToggleSettings.tsx

```diff
@@ -0,0 +1,8 @@
+import { MenuToggle, Flex } from '@patternfly/react-core';
+
+export const MenuToggleSettings: React.FunctionComponent = () => (
+  <Flex>
```

Comment:
nit - just an inconsistency, but I noticed the updated custom icon example puts `{' '}` between the items for a gap instead of wrapping them in `<Flex />`.